build/commands/lib/config.js

```diff
@@ -95,6 +96,8 @@ const parseExtraInputs = (inputs, accumulator, callback) => {
 }
 
 const Config = function () {
+  // TODO(yannic): Should the user be able to override this?
+  this.buildId = crypto.randomUUID()
```

Comment:
I think we should name it `gomaBuildId` and put it right after `this.defaultGomaJValue`.

Reply:
done